packages/aws-cdk-lib/awslint.json

```diff
@@ -886,6 +886,35 @@
     "props-physical-name:aws-cdk-lib.aws_apigateway.ResourceProps",
     "props-physical-name:aws-cdk-lib.aws_appsync.SourceApiAssociationProps",
     "docs-public-apis:aws-cdk-lib.aws_backup.Schedule",
-    "props-default-doc:aws-cdk-lib.aws_appsync.SourceApi.description"
+    "docs-public-apis:aws-cdk-lib.aws_appflow.CfnFlow.SAPODataPaginationConfigProperty.maxPageSize",
```

Comment:
why does the diff here contain linter rules that are irrelevant to apigatewayv2?

Reply:
@kaizencc good catch 😟 just removed those!